source/sass/components/publication-page.scss

```diff
@@ -14,9 +14,8 @@
 .publication-hero-container {
   padding-top: 137px;
   padding-bottom: 184px;
-  background-size: cover !important;
-  background-position: center center;
-  background: linear-gradient(
+  background-size: cover;
+  background-image: linear-gradient(
```

Comment:
along with changes in network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/publication_hero.html